CONTRIBUTING.md

```diff
@@ -36,6 +36,38 @@ For the moment it would not seem appropriate for our plugin to throw instances c
 of the [Error class](https://api.dart.dev/stable/2.4.0/dart-core/Error-class.html), as this is perhaps more
 appropriate for programmer's failures on the Dart side of things. But time will tell.
 
+### Push Notifications 
+
+#### Debugging the app from launch
```

Comment:
It would be really nice to have screenshots for both iOS and Android here!

Reply:
3765af68a76229b6c66afe1e975c815d598df9a4